### ISSUE TITLE
[Enhancement] Files commited with CRLF line endings

### ISSUE THREAD

**Issue by lpalashevski** [odpi/egeria#7415]
### Please describe the new behavior that that will improve Egeria

While working on PR conflicts IntelliJ showed me Line Separator Warning like this:

![Screenshot 2023-02-20 at 17 34 52 copy](https://user-images.githubusercontent.com/26764383/220166626-0a819253-a05e-4ba6-a1d5-7dfe87ecaf59.jpg)

For now committed as-is but this triggered me to check what we have across this git repo and this is the result:

```
> git ls-files --eol | grep "w/crlf"
i/crlf  w/crlf  attr/                 	Community-Guide.md
i/crlf  w/crlf  attr/                 	gradlew.bat
i/crlf  w/crlf  attr/                 	open-metadata-implementation/adapters/open-connectors/data-store-connectors/file-connectors/csv-file-connector/src/test/resources/ComplexColumnsWithColumnNames.csv
i/crlf  w/crlf  attr/                 	open-metadata-implementation/adapters/open-connectors/data-store-connectors/file-connectors/csv-file-connector/src/test/resources/NoColumnNames.csv
i/crlf  w/crlf  attr/                 	open-metadata-implementation/adapters/open-connectors/data-store-connectors/file-connectors/csv-file-connector/src/test/resources/SimpleColumnsWithColumnNames.csv
i/crlf  w/crlf  attr/                 	open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/CompDir-ContactEmail.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/CompDir-ContactList.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/CompDir-ContactPhone.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/EmplSAnl-EmpSalaryAnalysis.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/Employee-Dept.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/Employee-Employee.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/Location-WorkLocation.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/data-files/Patient-Patient.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/ldap/02-users.ldif
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/ldap/03-groups.ldif
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-samples/access-services-samples/asset-management-samples/CompanyDirectoryDatabase.csv
i/crlf  w/crlf  attr/                 	open-metadata-resources/open-metadata-samples/access-services-samples/asset-management-samples/ContactList.csv
```

Not worried about most of the files as it may be relevant to have CRLF (i.e. in bat file) but the question is do we standardize this?

At minimum we should convert the .java source file to use LF for consistency.

open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java

thoughts?

**Comment by planetf1** [comment]
I'm not sure how the git client will handle windows, but the 'gradlew.bat' has been created by the official gradle wrapper, so I would be inclined to leave it as is. We would never expect any manual edits of that file

When the wrapper is updated, it will again create a new .bat file -- for windows users

I do think the others should be changed.

**Comment by lpalashevski** [comment]
Thanks I am thinking only to change the .java file for consistency.

**Comment by lpalashevski** [comment]
Closing. Good enough fix #7650
